experimental/gittuf/root.go

```diff
@@ -935,6 +917,41 @@ func (r *Repository) RemovePropagationDirective(ctx context.Context, signer ssli
 	return r.updateRootMetadata(ctx, state, signer, rootMetadata, commitMessage, options.CreateRSLEntry, signCommit)
 }
 
+// ListPropagationDirectives lists all propagation directives defined in the
+// root metadata.
+func (r *Repository) GetPropagationDirectives(ctx context.Context, signer sslibdsse.SignerVerifier, signCommit bool) ([]tuf.PropagationDirective, error) {
```

Comment:
```suggestion
// GetPropagationDirectives lists all propagation directives defined in the
// root metadata.
func (r *Repository) GetPropagationDirectives(ctx context.Context, signer sslibdsse.SignerVerifier, signCommit bool) ([]tuf.PropagationDirective, error) {
```

The comment should match the function